tools/wake/main.cpp

```diff
@@ -582,6 +601,12 @@ int main(int argc, char **argv) {
     }
   }
 
+  if (label) {
+    std::string glob = argv[1];
+    std::replace(glob.begin(), glob.end(), '*', '%');
```

Comment:
I'm slightly skeptical of this line TBH but its probably fine. I saw land it and we can just say "oh darn" if we see a bug in the future here. 

We should add `?` being replaced with `_` as well btw since that's the single character setup for SQL

Reply:
> We should add ? being replaced with _ as well btw since that's the single character setup for SQL
want me to add that before landing?

Reply:
Yeah I don't see why not.